### PR TITLE
Update tag autofocus and sort stuff and string util

### PR DESCRIPTION
**Pull request by Rinder5**

### CODE REVIEW

File: src/main/java/seedu/toluist/controller/SortController.java [hunk 1]
```diff
@@ -21,10 +22,15 @@
 public class SortController extends Controller {
     private static final String RESULT_MESSAGE = "List is now sorted by: %s.";
     private static final String ERROR_MESSAGE = "Unable to sort by: %s";
+    private static final String NO_VALID_KEYWORD_MESSAGE = "No valid keyword entered."
```
Comment (louislai) [inline src/main/java/seedu/toluist/controller/SortController.java]: `MESSAGE` should be in front to follow the coding standard. Make sure you have everything changed

File: src/test/java/guitests/SortCommandTest.java [hunk 2]
```diff
@@ -1,36 +1,139 @@
...
     @Test
-    public void sort_multipleTimes() {}
+    public void sort_byInvalidParameter() {
+        commandBox.runCommand("sort a");
+        assertResultMessage("No valid keyword entered. Please type 'help sort' for details");
+    }
+
+    @Test
+    public void sort_byMultipleParametersIncludingDefault() {
+        commandBox.runCommand("sort priority default");
```
Comment (louislai) [inline src/test/java/guitests/SortCommandTest.java]: `runCommandThenCheckForResultMessage`

Comment (louislai) [reply src/test/java/guitests/SortCommandTest.java]: make sure the rest are changed as well

**Comment by louislai** [comment]
Unit tests for sorting?

File: src/test/java/guitests/SortCommandTest.java (hunk 2 above)
Comment (Rinder5) [reply src/test/java/guitests/SortCommandTest.java]: ?

**Pull request merged**